website/docs: Fix websocket default config for nginx proxy manager

MahmoudAlyuDeen opened the pull request:
## Details

Update suggested configuration replacing ~~commenting out~~  lines that cause Nginx Proxy Manager host to go offline.

Resolves:
- https://github.com/goauthentik/authentik/issues/10010
- https://github.com/goauthentik/authentik/discussions/7323
- https://github.com/goauthentik/authentik/issues/11453
- https://www.reddit.com/r/Authentik/comments/1c5sf6l/authentik_with_nginx_proxy_manager_not_possible/

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

MahmoudAlyuDeen committed:
Comment out problematic config at _nginx_proxy_manager.md

Resolves:
- https://github.com/goauthentik/authentik/issues/10010
- https://github.com/goauthentik/authentik/discussions/7323
- https://github.com/goauthentik/authentik/issues/11453
- https://www.reddit.com/r/Authentik/comments/1c5sf6l/authentik_with_nginx_proxy_manager_not_possible/

Signed-off-by: Mahmoud AlyuDeen <mahmoudalyudeen@gmail.com>

BeryJu reviewed (commented):
Aside from being baffled how this standard nginx configuration breaking NPM, if anything I dont think it should be commented out but rather removed, as it being commented out without a reason makes it even more confusing

MahmoudAlyuDeen committed:
Add working websocket configuration for nginx-proxy-manager.

Signed-off-by: Mahmoud AlyuDeen <mahmoudalyudeen@gmail.com>

MahmoudAlyuDeen commented:
> if anything I dont think it should be commented out but rather removed

Makes sense. I've replaced the current setup with a working and tested `websocket` setup - and kept it commented out by default.

I updated the PR title and description accordingly.

BeryJu committed:
Merge branch 'main' into patch-1

BeryJu committed:
remove commented out settings

Signed-off-by: Jens Langhammer <jens@goauthentik.io>
